ocrd_models/ocrd_page_user_methods/get_AllTextLine.py

```diff
@@ -0,0 +1,9 @@
+def get_AllTextLine(self):
+    """
+    Return all the TextLine in the document
+    """
+    ret = []
+    for reg in self.get_AllRegions(['Text']):
+        ret += reg.get_TextLine()
+    return ret
+
```

Comment:
Besides the plural, I recommend also offering an `order` kwarg here, delegating to `get_AllRegions` at the region level, and additionally respecting `textLineOrder` at the line level.

Reply:
4def1d97, implemented a `region_order` kwarg delegating to `order` in `get_AllRegions`. As for the `textLineOrder`, I've asked @tboenig to clarify what this means and how we can test it.

Reply:
> As for the `textLineOrder`, I've asked @tboenig to clarify what this means and how we can test it.

Ah, great – I was just going to ask you for that, cf. [this comment](https://github.com/OCR-D/core/pull/559/files/4bf12fb7eaa39fc2a73874df3b69c4b6b24b5965..4def1d974e9645753db924b5ca0c5ed1e7682ed5#r551472862)

Reply:
cf. https://github.com/PRImA-Research-Lab/PAGE-XML/issues/26